src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -83,6 +83,120 @@ public static class DnnCatalog
             return new DnnRetrainEstimator(env, options, DnnUtils.LoadDnnModel(env, modelPath, true));
         }
 
+        /// <summary>
+        /// The options for the <see cref="ImageClassificationTransformer"/>.
+        /// </summary>
+        public sealed class Options
+        {
+            /// <summary>
+            /// The names of the model inputs.
+            /// </summary>
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The names of the model inputs", ShortName = "inputs", SortOrder = 1)]
+            public string FeaturesColumnName;
+
+            /// <summary>
+            /// The name of the label column in <see cref="IDataView"/> that will be mapped to label node in TensorFlow model.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Training labels.", ShortName = "label", SortOrder = 4)]
+            public string LabelColumnName;
+
+            /// <summary>
+            /// The names of the requested model outputs.
+            /// </summary>
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The name of the outputs", ShortName = "outputs", SortOrder = 2)]
+            public string ScoreColumnName = "Score";
+
+            /// <summary>
+            /// Name of the tensor that will contain the predicted label from output scores of the last layer when transfer learning is done.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Argmax tensor of the last layer in transfer learning.", SortOrder = 15)]
+            public string PredictedLabelColumnName = "PredictedLabel";
+
+            /// <summary>
+            /// Validation set.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Validation set.", SortOrder = 15)]
+            public IDataView ValidationSet = null;
```

Comment:
Reference types are `null` by default. You don't need to explicitly put `= null` at the end here, and below.